README.md

```diff
@@ -165,23 +175,85 @@ In Device Simulator Express, you can use keyboard to interact with the device:
+
+1. Plug in your BBC micro:bit.
+2. Run the command `"Device Simulator Express: Open Serial Monitor"`.
+3. Select your baud rate for the serial port.
+4. The `print()` statements in your code will show in the output console.
+
+#### VII. Use the sensors in the Device Simulator Express
+
+Generating input for the sensors can be done by interacting directly with device on the webview
+or by using the toolbar.
+
+- **Push buttons:** click directly on the buttons on the device or use the keybindings.
+- **Temperature sensor, Light sensor, Acceleration sensor:** click on the corresponding button in the toolbar and change the value using the slider or the input box attached to it.
+
+#### VIII. Debug your project on the simulator
+
+1. Add breakpoints in your code
+2. Press F5 to enter the debugging mode, and you can start debugging line by line!
+
+### Commands
 Using the simulator for the micro:bit is similar to using the one for the CPX. The only difference is that the commands in the command palette display `Device Simulator Express: [micro:bit] <command>` instead of `Device Simulator Express: [Circuit Playground Express] <command>`. Currently, we support the following commands for micro:bit:
+- `Device Simulator Express: Getting Started`: Opens a page in VS Code that helps users get started with the extension and code they can write to play with the simulators.
+- `Device Simulator Express: Run Simulator`: Runs python code on the simulator.
 - `Device Simulator Express: [micro:bit] Open Simulator`: Opens an unsaved .py file with template code, also opens the simulator.
-- `Device Simulator Express: [micro:bit] New File`: Opens the simulator in the webView
-
-Please review the CPX's ["How to use" guide](#How-to-use) for more info.
+- `Device Simulator Express: [micro:bit] New File`: Opens the simulator in the webView.
+- `Device Simulator Express: [micro:bit] Deploy to Device`: Copies the current file to the micro:bit if the device is detected.
+- `Device Simulator Express: Open Serial Monitor`: Opens the serial monitor in the integrated output window.
+- `Device Simulator Express: Close Serial Monitor`: Stops the serial monitor and releases the serial port.
+- `Device Simulator Express: Change Baud Rate`: Changes the baud rate of the selected serial port. For BBC micro:bit, the default baud rate is 115200.
+- `Device Simulator Express: Select Serial Port`: Changes the current serial port.
 
 ### Keybindings
 - Push Button `A for A, B for B, C for A & B`
 - Refresh the simulator: `SHIFT + R`
+- Run the simulator: `SHIFT + F`
 
 ## Contribute
-[See here for steps to run the extension locally.](/docs/developers-setup.md)
+[See here for steps to run the extension locally.](https://github.com/microsoft/vscode-python-devicesimulator/blob/dev/docs/developers-setup.md)
 
 ## Provide feedback
 
-To report issues, provide feedback or requests, please use this link: [Provide Feedback](https://aka.ms/AA5xpxx).  

```

Comment:
Maybe we can add a line also encouraging them to leave a review on the marketplace? The nature of github's issue section is more for technical issues, whereas maybe we should publicize a place for general comments.

Reply:
good idea